### PR TITLE
Client setup

### PR DESCRIPTION
**Pull request by joshskkim**
@Team-Gandalf/dope 
Initial client setup with components and webpack

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "seed": "node db/faker.js",
-    "server": "nodemon server -i client",
+    "server": "nodemon server/server.js -i client",
```
Comment (sajjanrajvaidya) [inline package.json:9]: What is '-i client' meant to be doing here?

Comment (joshskkim) [reply package.json:9]: it ignores the client so it doesn't keep refreshing the server when I change stuff on the client

File: client/src/components/Reviews.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React, { useState, useEffect, useCallback } from 'react';
```
Comment (sajjanrajvaidya) [inline client/src/components/Reviews.jsx:1]: Fantastic!

**Review by sajjanrajvaidya — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (sajjanrajvaidya) [reply package.json:9]: Great. I'll be doing the same.
Thank you.

**Pull request merged**